velum/impl/bot.py

```diff
@@ -27,7 +27,7 @@
 _LOGGER = logging.getLogger("velum.bot")
```

Comment:
This should be renamed to `"velum.client"`

Reply:
:clown_face: